beet/src/beets/enums.ts

```diff
@@ -24,24 +24,29 @@ export function fixedScalarEnum<T>(
 ): FixedSizeBeet<Enum<T>, Enum<T>> {
   return {
     write(buf: Buffer, offset: number, value: T) {
-      const idx = Object.values(enumType).indexOf(value)
-      if (idx < 0) {
+      // @ts-ignore
+      if (enumType[value] === undefined) {
```

Comment:
Is there no way to do this without `@ts-ignore`? I tried very hard to make all types check out and don't think I use it anywhere (only while I was prototyping).